src/Drupal8/Rector/Deprecation/DBRector.php

```diff
@@ -61,7 +61,7 @@ class DBRector extends AbstractRector implements ConfigurableRectorInterface
     private array $configuration;
 
     /**
-     * @var \DrupalRector\Services\AddCommentService
+     * @var AddCommentService
```

Comment:
My understanding is that @var descriptions should use the full class path, so I don't understand these changes.

Reply:
This is part of the Symfony code standards. https://cs.symfony.com/doc/rules/import/fully_qualified_strict_types.html